crates/controller/src/hud.rs

```diff
@@ -79,3 +102,32 @@ fn process_events(
         }
     }
 }
+
+pub fn spawn_hud(
+    mut commands: Commands,
+    text_props: Res<TextProps>,
+) {
+    let font = text_props.as_ref();
+    commands.spawn(
+        NodeBundle {
+            style: Style {
+                justify_content: JustifyContent::SpaceBetween,
+                flex_direction: FlexDirection::Row,
+                align_items: AlignItems::FlexEnd,
+                position_type: PositionType::Absolute,
+                size: Size {
+                    width: Val::Percent(100.0),
+                    height: Val::Percent(100.0),
+                },
+                ..Style::default()
+            },
+            background_color: BackgroundColor::from(Color::NONE),
```

Comment:
`Color::NONE.into()` would be more concise & idiomatic in this context (the type is derived from `NodeBundle` struct).